project/src/hooks/use-add-review-state.ts

```diff
@@ -0,0 +1,18 @@
+import {ChangeEvent, useCallback, useState} from 'react';
+
+type RatingType = (event: ChangeEvent<HTMLInputElement>) => void;
+type ReviewType = (event: ChangeEvent<HTMLTextAreaElement>) => void;
+
+const useAddReviewStates = (): [RatingType, ReviewType, boolean, string, string] => {
+  const [comment, setReview] = useState<string>('');
+  const [rating, setRating] = useState<string>('');
+
+  const isFormInvalid = Boolean(rating === undefined || comment.length < 50);
```

Comment:
```suggestion
  const isFormInvalid = Boolean(rating === undefined || comment.length < 50);
```
а тут точно рейтинг может быть undefined?

добавь константу на рейтинг